e2e/tests/navigation.test.ts

```diff
@@ -29,21 +29,17 @@ import { createTestHttpServer } from './testServer';
 const electronService = new ElectronServiceFactory();
 
 let server: Server;
-let hostname: string;
-let port: number;
 let url: string;
 
-beforeEach(() => {
-  const { server: s, hostname: h, port: p } = createTestHttpServer();
+beforeAll(async () => {
+  const { server: s, port } = await createTestHttpServer();
```

Comment:
Start startup/teardown code was not setup correctly, this would lead to test starting before the server starts, Fixed this as well.